R/theme-elements.r

```diff
@@ -13,6 +13,8 @@
 #' @param fill Fill colour.
 #' @param colour,color Line/border colour. Color is an alias for colour.
 #' @param size Line/border size in mm; text size in pts.
+#' @param inherit.blank Should this element inherit the existence of an
+#' element_blank among its parents?
```

Comment:
Can you expand on this a little bit? I assume if it's `TRUE` the "blank-ness" will be inherited, otherwise it will skip the parent and look at the grand parent?